README.md

```diff
@@ -1 +1,16 @@
-# flink-sql
\ No newline at end of file
+# Flink SQL Runner
+
+An application to execute Flink SQL jobs.
+
+## Building and running Flink SQL Runner
+
+1. Build application
+    ```
+    mvn package
+    ```
+2. Build an image
+    ```
+    minikube image build flink-sql-runner -t flink-sql-runner:latest
+    ```
+
+See /examples directory to see how to run FlinkDeployment using the SQL runner.
```

Comment:
Are there supposed to be further instructions about setting up the environment? (non-blocking, can defer to another PR)

Reply:
Yes we can do that in another PR certainly, as we will add install directory later.